config/initializers/react_on_rails.rb

```diff
@@ -36,7 +36,7 @@
 
   config.logging_on_server = true # Default is true. Logs server rendering messags to Rails.logger.info
 
-  config.raise_on_prerender_error = false # change to true to raise exception on server if the JS code throws
+  config.raise_on_prerender_error = true # change to true to raise exception on server if the JS code throws
```

Comment:
Do we want to make this global decision?
While developing I have used:
```
= react_component("OnboardingGuideApp", props: props, prerender: true, raise_on_prerender_error: true)
```
But I'm not sure I want to keep it (or at least switch it always on)